adalflow/adalflow/components/model_client/azureai_client.py

```diff
@@ -0,0 +1,450 @@
+"""AzureOpenAI ModelClient integration."""
+
+import os
```

Comment:
This feels almost the same as openai client, maybe we should just subclass from openai client and overwrite a few functions.